### DISCUSSION TITLE
How use uno wifi r3 atmega328p esp8266

### DISCUSSION THREAD

**Discussion by AL2ZD** [revoxhere/duino-coin#1069] [Q&A]
Con me use uno wifi r3 atmega328p esp8266 without pc?

**Comment by gsrbr** [comment]
Hello @AL2ZD are you talking about this board?
![image](https://user-images.githubusercontent.com/27559775/138187331-9edc36c6-98c0-4d18-8784-c459a64a8086.png)
Honestly I've never seen anyone mine with a board like this... as they are not so common in the market.... But I believe it is not impossible to make her mine via wifi... but I believe this code doesn't exist yet. ... I don't know how it works, but maybe it's easier to mine for her esp8266, instead of using atmega328p ... I've never really touched one of these
